src/Toolkit/Toolkit.WPF/UI/Controls/BasemapGallery/BasemapGallery.Theme.xaml

```diff
@@ -110,6 +110,7 @@
             <Setter.Value>
                 <DataTemplate>
                     <StackPanel
+                        Margin="5"
```

Comment:
The style shouldn't change with new margins. Please try and keep the existing layout the same and only add the 3D icon.